app/controllers/application_controller.rb

```diff
@@ -11,4 +12,13 @@ def recent_posts(id)
     user = User.find_by(id:)
     user.first_three_posts
   end
+
+  protected
+
+  def update_allowed_parameters
+    devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:name, :surname, :email, :password) }
+    devise_parameter_sanitizer.permit(:account_update) do |u|
+      u.permit(:name, :surname, :email, :password, :current_password)
+    end
```

Comment:
- [ ] Kindly use `:password_confirmation` instead of `:current_password`  as it is the correct input name that you are using in the view at `/app/views/devise/registrations/new.html.erb`